custom-vu/src/main/kotlin/jces1209/vu/ScenarioSimilarities.kt

```diff
@@ -89,6 +91,11 @@ class ScenarioSimilarities(
             meter = meter,
             scrumBoardsMemory = boardsMemory.scrum,
             jiraTips = JiraTips(jira.driver)
+        ),
+        workOnTopBar = WorkOnTopBar(
```

Comment:
Adding it on the end increases risk of Git conflict -> unnecessary work.
Worth keeping in mind for the next time.